cdk/cfn.yaml

```diff
@@ -14,7 +14,7 @@ Parameters:
     InstanceSize:
         Type: String
         Default: small
-    VpcId:
+    OldVpcId:
```

Comment:
We updated this name as it was conflicting with a parameter coming from `@guardian/cdk`